src/rational.rs

```diff
@@ -113,6 +130,14 @@ impl Rational {
     }
 
     /**
+     * Returns this Rational to the nearest Int
+     */
+    pub fn round(&self) -> Int {
+        let r = self + Rational::new(self.sign().into(), 2.into());
```

Comment:
Operations with small integers are implemented more efficiently than with full-blown arbitrary-precision ints. Also the lack of allocation is a decent performance saver.